bin/regression-wally

```diff
@@ -363,6 +370,7 @@ os.chdir(regressionDir)
 coveragesim = "questa"  # Questa is required for code/functional coverage
 #defaultsim = "questa"   # Default simulator for all other tests; change to Verilator when flow is ready
 defaultsim = "verilator"   # Default simulator for all other tests
+buildrootbootsim = "questa"
```

Comment:
Might be better to define a lockstepsim and use it both for the lockstep buildroot boot and all of the lockstep tests by directory

Reply:
sure, i can do that.

Reply:
done